bjoern/py2py3.h

```diff
@@ -30,6 +31,18 @@
 #define _Bytes_Resize(obj, len) _PyString_Resize(obj, len)
 #define _FromLong(n) PyInt_FromLong(n)
 #define _Unicode_EncodeLatin1(u) (Py_INCREF(u),u)
+
+#ifdef __GNUC__
+static PyObject *_Unicode_Concat(PyObject *l, PyObject *r) __attribute__ ((unused));
+#endif
+static PyObject *_Unicode_Concat(PyObject *l, PyObject *r) {
+    PyObject *ret = l;
+
+    Py_INCREF(l);  /* reference to old left will be stolen */
+    PyString_Concat(&ret, r);
+
+    return ret ? ret:NULL;
```

Comment:
That's a noop right?

Reply:
It was inspired by https://github.com/encukou/py3c/blob/c3dc22f9d8652ef0874f65bb9e54e6c4ff9371e1/include/py3c/compat.h#L83
I was not sure, but I think too that it is a noop, so i drop it.